docker/quickstart/myaac/bootstrap.php

```
@@ -0,0 +1,286 @@
+	$account = new OTS_Account();
+	$account->find($adminAccount);
+
+	if (!$account->isLoaded()) {
+		$account->create($adminAccount);
+	}
+
+	$account->setPassword(encrypt($adminPassword));
+	$account->setEMail($adminEmail);
+	$account->save();
+	$account->setCustomField('created', time());
+	$account->setCustomField('web_flags', FLAG_ADMIN + FLAG_SUPER_ADMIN);
```

PHPStan (2.1.54)

[error] Constant FLAG_ADMIN not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)